sprint0/sprint0/Items/ItemSpawner.cs

```diff
@@ -11,21 +11,21 @@ public class ItemSpawner
 
         private readonly Random rand = new Random();
         private readonly Dictionary<string, string[]> enemyItemMap = new Dictionary<string, string[]> {
-            {"Gel", new string[] { "", "heart container", "rupee", "key" } },
-            {"Goriya",new string[] { "" } },
-            {"Keese",new string[] { "" } },
-            {"Snake",new string[] { "" } },
-            {"Stalfos",new string[] { "" } },
-            {"Wallmaster",new string[] { "" } },
+            {"Gel", new string[] { "", "arrow", "rupee"} },
+            {"Goriya",new string[] { "","rupee" , "bomb"} },
+            {"Keese",new string[] { "", "rupee", "meat" } },
+            {"Snake",new string[] { "", "raft", "stepladder", "rupee" ,"fairy"} },
+            {"Stalfos",new string[] { "", "rupee", "clock", "potion", "bomb" , "meat"} },
```

Comment:
- for the potions you might have to specify if it's the blue or red potion.
- Also there might be discrepancy with the food. it's called meat in some places but the official name (which is what the enum has) calls it food. so we'll have to go back and double check that